searchcommon/src/vespa/searchcommon/attribute/basictype.h

```diff
@@ -27,13 +27,10 @@ class BasicType
         MAX_TYPE
     };
 
-    explicit
-    BasicType(int t) : _type(Type(t)) { }
-    explicit
-    BasicType(unsigned int t) : _type(Type(t)) { }
+    explicit BasicType(int t) : _type(Type(t)) { }
+    explicit BasicType(unsigned int t) : _type(Type(t)) { }
     BasicType(Type t) : _type(t) { }
-    explicit
-    BasicType(const vespalib::string & t) : _type(asType(t)) { }
+    explicit BasicType(const vespalib::string & t) : _type(asType(t)) { }
```

Comment:
Please don't mix style changes (reformatting) with functional changes.

Reply:
Moved out to #6627